code/modules/gear_presets/wy.dm

```diff
@@ -63,13 +63,42 @@
 	rank = JOB_EXECUTIVE_SPECIALIST
 	paygrade = PAY_SHORT_WYC5
 
+/datum/equipment_preset/wy/exec_spec/lawyer
+	name = "Corporate - E - Lawyer"
+
+/datum/equipment_preset/wy/exec_spec/lawyer/load_gear(mob/living/carbon/human/new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/distress/pmc(new_human), WEAR_L_EAR)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/liaison_suit/blue(new_human), WEAR_BODY)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/jacket/marine/corporate/blue(new_human), WEAR_JACKET)
+	new_human.equip_to_slot_or_del(new /obj/item/tool/pen/clicky(new_human), WEAR_IN_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/clipboard(new_human), WEAR_IN_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/notepad/blue(new_human), WEAR_R_STORE)
+	new_human.equip_to_slot_or_del(new /obj/item/device/taperecorder(new_human), WEAR_L_STORE)
+
+	..()
+
 /datum/equipment_preset/wy/exec_supervisor
 	name = "Corporate - F - Executive Supervisor"
 	flags = EQUIPMENT_PRESET_EXTRA
 	assignment = JOB_EXECUTIVE_SUPERVISOR
 	rank = JOB_EXECUTIVE_SUPERVISOR
 	paygrade = PAY_SHORT_WYC6
 
+/datum/equipment_preset/wy/exec_supervisor/lawyer
+	name = "Corporate - F - Lawyer"
+
+/datum/equipment_preset/wy/exec_supervisor/lawyer/load_gear(mob/living/carbon/human/new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/distress/pmc(new_human), WEAR_L_EAR)
```

Comment:
Same

Reply:
Sure, mostly gave them the PMC one to track the CL and have access to command chat.